src/deployment/android.md

```diff
@@ -95,64 +95,66 @@ To find out the latest version, visit [Google Maven][].
 +<style name="NormalTheme" parent="Theme.MaterialComponents.DayNight.NoActionBar">
 ```
 
-## Signing the app
+## Sign the app
```

Comment:
Since this page has existed for a while, there may be links to the old fragment. You can add a backup anchor for the old links like this:

```suggestion
<a id="signing-the-app"></a>
## Sign the app
```